unblob/file_utils.py

```diff
@@ -234,6 +234,12 @@ def iterate_file(
         yield data
 
 
+def copy_to_file(input_file: File, output_path: Path, offset: int, length: int):
```

Comment:
There is a `carve_chunk_to_file` in `extractor.py` that does something like this, except it takes `Chunk` instead of `offset` and `length` and most importantly creates the parent directory if it is missing!

`copy_to_file` is fine in `file_utils`, but then the `carve_chunk_to_file` should go.